src/components/nav/AppShell.tsx

```diff
@@ -8,42 +8,44 @@ import {
   Text,
   Box,
   type BoxProps,
-  type LinkProps,
-  Link,
+  Link as ChakraLink,
   Flex,
 } from "@chakra-ui/react";
 import Head from "next/head";
+import Link, { type LinkProps } from "next/link";
 import { BsGithub, BsPersonCircle } from "react-icons/bs";
 import { useRouter } from "next/router";
 import { type IconType } from "react-icons";
-import { RiFlaskLine } from "react-icons/ri";
+import { RiDatabase2Line, RiFlaskLine } from "react-icons/ri";
 import { signIn, useSession } from "next-auth/react";
 import UserMenu from "./UserMenu";
 
-type IconLinkProps = BoxProps & LinkProps & { label?: string; icon: IconType };
+const SHOW_DATA = false;
```

Comment:
Could drive this with an env variable so you don't have to remember to set it to false before committing each time

Reply:
Agreed. I don't think it will be very long before we add this to our production, but I'll add an env variable in the meantime